### PR TITLE
Organization editing

### PR DESCRIPTION
**Pull request by zpnk**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -30,6 +31,10 @@ if (process.env.NODE_ENV === 'development') {
 
 app.use(express.static(contentBase));
 
+app.use('/s3', s3router({
+  bucket: process.env.AWS_S3_BUCKET,
```
Comment (zpnk) [inline src/server.js]: Yes. `s3router` uses the `aws-sdk` under the hood, which automatically reads in the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.

Comment (amccloud) [inline src/server.js]: Ah okay. If you resolve conflicts we can merge this.

**Comment by zpnk** [comment]
Conflicts resolved!

**Comment by zpnk** [comment]
This has been rebased against the migration config changes that landed in #9. Ready to merge.

**Pull request merged**